### ISSUE TITLE
Excluding `day` and `month` values from interpreted records

### ISSUE THREAD

**Issue by NgAMB** [gbif/portal-feedback#4916]
I am wondering about occurrences with a provided `day` and `month` where those values have been excluded from the interpreted record in occurrence.txt in the DwC-A download.
e.g.
https://www.gbif.org/occurrence/1262187452
https://www.gbif.org/occurrence/2859457641
Data in these fields can still be useful even if the `year` is not provided. I know now that I can recover these values from the verbatim.txt file, but I had to know they existed in the first place before I could go looking for them. Unlike some other problems (e.g. invalid date, improbable date) there was no flag indicating this exclusion in the `issue` field.

I am particularly wondering about their exclusion because of instances like the first example, which shows the following for the Event:

<html><body>

Term | Interpreted | Original | Remarks
-- | -- | -- | --
Day |   | 12 | Excluded
Month |   | 8 | Excluded
Start day of year | 44.643036 | 44.643036 |  

</body>
</html>

It appears that a `decimalLatitude` value has slipped into this data provider's `startDayOfYear` field, which is clearly not the fault of GBIF. What I am wondering, though, is why this value was retained while the provided `day` and `month` -- which would allow one to calculate `startDayOfYear` -- are excluded. Would it not be better to include a flag for an apparent mismatch, similar to the `RECORDED_DATE_MISMATCH` flag? Or, in this particular case, retain the valid `month`-`day` combination and exclude the `startDayOfYear` value that specifies a particular second?

What overall is the rule/reasoning behind excluding `day` and `month` values? Why retain `startDayOfYear` while excluding `day` and `month`? Or does this situation occur to such a small percentage of records that it isn't explicitly handled?

**Comment by Mesibov** [comment]
@NgAMB, I'm sure you've noticed this already, but the herbarium sheet label for that occurrence says "12 August 1957". Further, the Field Museum's online catalog has "Date collected: 12 August 1957" (https://collections-botany.fieldmuseum.org/catalogue/3364255). Notice also that the Field gives the catalogue number as "2268753", its EMu IRN as "3364255" and its barcode number as "V0388951F", but it's the barcode number that appears in the GBIF record as catalog number. The Field's conversion of EMu records to Darwin Core may be the problem here.

**Comment by NgAMB** [comment]
Yes, it seems like in that case there is something going wrong on The Field's end. I'm not so much wondering about that record, as I am interested in what GBIF's heuristic is in cases like it, and possibly improving that heuristic if I've stumbled across an edge case that it would be useful to handle.

The "Remarks" column on an individual occurrence webpage is helpful in telling me what has been chosen, but it's not so convenient when I am working with a download containing tens of thousands of records (unless I am missing where this information is inside the DwC download?). It also doesn't tell me *why* that action has been taken. Perhaps the rule is such that, for my purposes, I should be using the verbatim file instead. Though that would rob me of the benefits GBIF provides in interpreting data -- like the inference of `day` `month` and `year` values for [this occurrence](https://www.gbif.org/occurrence/3351392203).  

And maybe I'm just a bit nosy and like to know what is going on "under the hood"...

**Comment by Mesibov** [comment]
@NgAMB, there is more than 1 hood. I did a scoping audit in 2021 of the herbarium records from which you give your 2nd example. In correspondence with the collection manager in charge of botany, I learned that the data at the time were going through Symbiota before harvesting by GBIF, and that Symbiota was evidently responsible for errors in dates that were not present in the original CM: Botany dataset.

Yes, there are benefits in getting data "interpreted" by aggregators. There are also costs, and end-users need to be cautious (https://doi.org/10.3897/zookeys.751.24791) when dealing with second- or third-hand data.
